docs/installation/local.md

```diff
@@ -28,13 +28,13 @@ venv\Scripts\activate # Windows
 pip install -r requirements.txt
 ```
 
-To set up MongoDB on your server : 
+To set up MongoDB on your server :
 
 ```bash
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
 echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
 sudo apt-get update
-sudo apt-get install -y mongodb-org
+sudo apt-get install -y mongodb
```

Comment:
I can find that with 
`sudo apt-get install -y mongodb-org`


`E: There were unauthenticated packages and -y was used without --allow-unauthenticated` 
comes have you tried this thing

Reply:
I didn't get this error. What I got with old version of command is the package was not located . The I checked the link mentioned on the issue page where it says to remove the org. And when I tried that while installing, it worked (removing org works )